src/store/actions/transactions.js

```diff
@@ -107,22 +112,44 @@ export const transactionCreated = data => async (dispatch, getState) => {
     account: account.info[activeToken],
     passphrase: account.passphrase,
     network,
+    isHwSigning: account.loginType !== loginTypes.passphrase.code,
   };
 
-  const [error, tx] = account.loginType === loginTypes.passphrase.code
-    ? await to(create(params, activeToken))
-    : await to(signSendTransaction(account, data));
+  let [error, tx] = await to(create(params, activeToken));
+
+  if (error) {
+    dispatch({
+      type: actionTypes.transactionSignError,
+      data: error,
+    });
+    return;
+  }
+
+  if (params.isHwSigning) {
```

Comment:
I've explained this in `src/utils/api/transaction/lsk.js`. Since that's the utility function, I started the review from that file :-)